.evergreen/install-tools.sh

```diff
@@ -39,13 +39,10 @@ swiftenv local $SWIFT_VERSION
 
 if [ $1 == "swiftlint" ]
 then
-    build_from_gh swiftlint https://github.com/realm/SwiftLint "0.43.1"
+    build_from_gh swiftlint https://github.com/realm/SwiftLint "0.44.0"
 elif [ $1 == "swiftformat" ]
 then
-    build_from_gh swiftformat https://github.com/nicklockwood/SwiftFormat "0.47.3"
-elif [ $1 == "sourcery" ]
-then
```

Comment:
we don't actually use sourcery for this library, I think this was just copied from the driver.